README.md

```diff
@@ -362,3 +362,17 @@ Now with your team of miners working at full force, you lay back, relax, and enj
 | :-- |
 
 
+## Troubleshooting
```

Comment:
We need to scope and describe the issue more precisely. Suggestion:

> There is a known issue with this tutorial for MacOS users: in rare occurrences, when switching between the MAS console and the tutorial GUI, one of them might freeze. If you encounter this issue, we suggest disabling (...)

For consistency, make sure to add links to the relevant lines (just like we do in the rest of the README).